RunningOrder/Sprint/Managers/SprintManager.swift

```diff
@@ -60,6 +60,25 @@ final class SprintManager: ObservableObject {
         }
     }
 
+    func deleteSprint(_ sprint: Sprint) {
+        guard let index = self.sprints.firstIndex(of: sprint) else {
+            Logger.error.log("couldn't find index of sprint in stored sprints")
```

Comment:
Should we throw instead?

Reply:
not in the current implementation of the method : the errors found when deleting are just logged for now. If it changes, and we need to display error to users, then we will return a publisher to handle error in the view side :)